superbench/runner/runner.py

```diff
@@ -385,7 +388,23 @@ def __merge_monitor_metrics(self, node_path):
 
         return metrics_summary
 
-    def _run_proc(self, benchmark_name, mode, vars):
+    def _prepare_config(self, args):
```

Comment:
This function doesn't change args. What's the goal for this function?

Reply:
In current PR, is just used to validate the args for mpi-parallel mode. In next PR, it will be used to add `generate_ibstat` function and add the path of ibstatfile to args.